### ISSUE TITLE
Request: Do not generate (or at least have an option to disable) superfluous comments

### ISSUE THREAD

**Issue by daviddenton** [jmfayard/gradle-kotlin-dsl-libs#14]
It would be great to ditch the "up-to-date" messages. Kind of feels like superfluous info.

Also, if there could be an option not to generate the website information in the plugin, that would be ace.

thanks :)

**Comment by jmfayard** [comment]
> It would be great to ditch the "up-to-date" messages. Kind of feels like superfluous info.

very true, will do it :)

> Also, if there could be an option not to generate the website information in the plugin, that would be ace.

Mmh do we really need an option for that or is it rather that I should improve it like this https://github.com/ben-manes/gradle-versions-plugin/issues/266

![image](https://user-images.githubusercontent.com/459464/47641398-35c43b80-db66-11e8-8eaf-6fa7f9ffa815.png)

**Comment by daviddenton** [comment]
Re the website info, I think we'd like absolutely nothing in the versions file apart from the name, the version number, and the note that a new version is available.

One more idea - not that there is an easy way to to this ) - but it would also be great to avoid pulling in RC and alpha versions to the available tag. :)

**Comment by jmfayard** [comment]
Got it 

> One more idea - not that there is an easy way to to this ) - but it would also be great to avoid pulling in RC and alpha versions to the available tag. :)

actually it may be easy, have a look at https://github.com/ben-manes/gradle-versions-plugin#revisions
is that what you have in mind?

@daviddenton

**Comment by jmfayard** [comment]
@daviddenton Thanks a lot for your feedback, I released version `0.2.6` where `Versions.kt` is a lot cleaner

See
- https://github.com/jmfayard/gradle-kotlin-dsl-libs/blob/development/CHANGELOG.md
- https://github.com/jmfayard/http4k/blob/452df734caa6bbf4adb56b8a2f87fa52ec32dbd8/buildSrc/src/main/kotlin/Versions.kt

**Comment by daviddenton** [comment]
That's awesome - it's much cleaner now.

Interestingly, we do get 1 "exceeded" warning. Check it out...
```
    const val jmfayard_github_io_gradle_kotlin_dsl_libs_gradle_plugin: String =
            "0.2.6" // exceed the version found: 0.2.5
```

😂
